dotnet/year-service/Controllers/YearController.cs

```diff
@@ -16,25 +16,17 @@ public class YearController : ControllerBase
         [HttpGet]
         public async Task<int> GetAsync()
         {
-            using var activity = Startup.ActivitySource.StartActivity("DetermineYear");
+            using var activity = Startup.ActivitySource.StartActivity("🗓 get-a-year ✨");
             activity?.SetTag("banana", 1);
             var year = await DetermineYear();
             return year;
         }
 
         private static async Task<int> DetermineYear()
         {
-            await SleepAwhile();

```

Comment:
Opted not to make the spans longer than actual code takes to run.

Reply:
Understandable, though quick note: This is now an async method with no await.

Reply:
Yep, agree with @JamieDanielson - without an await, you can remove the async method modifier 👍🏻

Reply:
Removed all the async from Year since there is nothing to await.